### DISCUSSION TITLE
A Proposal for Filecoin Ecosystem Support Scheme

### DISCUSSION THREAD

**Discussion by sxxfuture-official** [filecoin-project/FIPs#944] [Enhancements - Governance]
# Proposer

SXX Future Data from Asia Storage Provider Working Group

# Background

In today’s blockchain world, the competition of public chains has largely evolved into the competition of chain’s  utility, performance and the strength of the ecosystem. The more mature the ecosystem is, the more use cases the chain will develop which in turn making it more dynamic. The launch of FEVM has opened the door to the ecosystem development of Filecoin, and various applications led by DeFi use cases have bloomed. However, compared with other public chains, the ecosystem development of Filecoin is very slow, lacking major infrastructures such as stablecoins, oracles and indexes. Developing DeFi on FEVM is drastically more costly on other EVM compatible chains. Protocol Labs(PL) and Filecoin Foundation(FF) with its vast network resources vested have inadequately supported this ecosystem.

# Proposal

1. **Establish an Ecosystem foresight service team:**  To plan the ecosystem development of Filecoin Network. Focus on supporting projects that have an important foundation or significant positions for the ecosystem compared to other chains. Publicly encourage and support various projects, and pour resources in development, marketing, investments, and collaboration from other chains into Filecoin. The ecosystem team should have longer term planning and deliver key milestones.
2. **Establish an ecosystem support fund:** Set up funds and increase the obligation to support ecosystem development. No less than 1/2 of holdings of PL and FF should be used to support Filecoin ecosystem projects, increase grants, and disclose the usage and progress information of each funded ecosystem project to the community every month;
3. **Setup Defi Ecosystem fund:** PL & FF need to set up a fund dedicated to the DeFi ecosystem. Making it more prosperous and solving sticky issues for projects building on top of. The fund size should be no less than 1/4 of the total holdings of PL and FF.
4. **Manage approved DeFi projects list Filecoin Foundation:** Vet and verify project’s validity and security for the DeFi community and users. PL & FF should stake at least 20%  each protocol’s staking pool of each project, and support each Defi project to connect token holders, SPs and educate users, market DeFi community in order to expand the network;
5. **Emergency DeFi rescue fund for liquidity crises:** There should be a new fund setup to provide security and stability to the DeFi project. When a project faces liquidity crises, the community can vote to unlock emergency liquidity to  rescue and inject deposits to restore the normal liquidity level. This should only apply to projects with top ten TVL. Gradually withdraw after liquidity is restored.

**Comment by 852031971** [comment]
support

**Comment by fil88888888** [comment]
同意

**Comment by fil88888888** [comment]
支持

**Reply by 205312327** [reply]
支持

**Comment by kongxinputi** [comment]
support

**Comment by toutuoge** [comment]
支持

**Comment by jingyu0212** [comment]
支持！

**Comment by HJann762** [comment]
支持

**Comment by michael8908** [comment]
Although FIL have the basic functions of FEVM, retrieval, and parts of computation, which act as important bases to support WEB3 DAPP especially for scaled DAPP, but the IPC (the key connection hub to integrate FEVM, retrieval, computation to an organic system) doesn't come out by now. It is the biggest pain point for not only DAPP entering FIL ecosystem but also attracting capital in. So, to focus on IPC development, test and launch, NOW, is the most urgent/crucial task for FILECOIN.

**Reply by ggyy2024** [reply]
Agree，but IPC & Ecosystem Support can proceed at the same time. No time to waste！

**Reply by jaychoucc** [reply]
Launch IPC on the main network as soon as possible

**Reply by 205312327** [reply]
尽快在主网上起动IPC

**Comment by Fatman13** [comment]
lfg

**Comment by ggyy2024** [comment]
support

**Comment by plutoyi** [comment]
支持

**Comment by JIANGRUILIN** [comment]
支持，并强烈要求Filecoin基金会和协议实验室的执行

**Reply by 205312327** [reply]
支持，并强烈要求Filecoin基金会和协议实验室的执行

**Comment by Ch248248** [comment]
支持三峡星的观点，也代表我们广大坚定持币的心声，以后有一个好的回报！！！！！！

**Reply by 205312327** [reply]
支持三峡星的观点，也代表我们广大坚定持币的心声，以后有一个好的回报！！！！！！

**Comment by sharkfee** [comment]
同意

**Comment by The-Wayvy** [comment]
I agree on prioritizing DeFi

We pursue pipe dreams like DeSci while not even having a stablecoin.

pick the low hanging fruit

**Comment by a351765899** [comment]
support

**Comment by Ch248248** [comment]
同意，大力支持

**Comment by xiaomdong** [comment]
support

**Comment by jaychoucc** [comment]
support

**Comment by gugege521** [comment]
支持该提案，希望社区的力量能够得到发挥，更希望fil团队可以加速技术的研发，带动币价起飞

**Comment by piaoliuping1** [comment]
同意

**Comment by AlanGreaterheat** [comment]
Support！！！

**Comment by Bllaaa** [comment]
支持
